common/scala/src/main/scala/whisk/core/containerpool/logging/ElasticSearchRestClient.scala

```diff
@@ -59,7 +59,8 @@ case class EsQueryBoolMatch(key: String, value: String)
 case class EsQueryOrder(field: String, kind: EsOrder)
 case class EsQuerySize(size: Integer)
 case class EsQueryAll() extends EsQueryMethod
-case class EsQueryMust(matches: Vector[EsQueryBoolMatch], range: Option[EsQueryRange] = None) extends EsQueryMethod
+case class EsQueryMust(matches: Vector[EsQueryBoolMatch], range: Option[Vector[EsQueryRange]] = None)
```

Comment:
Can `range` be "just" be a `Vector` defaulting to `empty`? Since the jsonformat is manual anyway, we can just omit the whole thing if the list is empty. Thoughts?